tests/lib/Asset/ThemePackageTest.php

```diff
@@ -18,17 +19,17 @@ class ThemePackageTest extends TestCase
     /**
      * @var \PHPUnit\Framework\MockObject\MockObject|\Ibexa\DesignEngine\Asset\AssetPathResolverInterface
      */
-    private $assetPathResolver;
+    private MockObject $assetPathResolver;
```

Comment:
And in the other cases here:
```suggestion
    private MockObject & AssetPathResolverInterface $assetPathResolver;
```